ci/print_revision_sql.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Get the latest migration from the file received as a parameter
+# and print it with escaped newlines
+set -e;
+
+
+migration_file=$1
```

Comment:
This won't work if a PR contains more than one new alembic revision :/

Reply:
I take the the latest one, always. But I think you're saying that I should take the first migration added in the PR and print the SQL from its `down_revision` to the latest `revision`. So the script would need 2 files as a parameter which are the same in case you add a single migration.

What's your suggestion?

Reply:
Alright, changed the script and this is the output for something old up the the present revision:

```bash
older_migration=newdle/migrations/versions/20200914_1149_679eab4aab01_add_comment_column_to_participants.py
migration_file=newdle/migrations/versions/20200922_1759_43f0bf77bf97_add_newdle_creator_email_notify.py    

./ci/print_revision_sql.sh $older_migration $migration_file 
```
Output:

```
INFO  [alembic.runtime.migration] Generating static SQL
INFO  [alembic.runtime.migration] Will assume transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade 93a638b96375 -> 679eab4aab01, Add comment column to participants
INFO  [alembic.runtime.migration] Running upgrade 679eab4aab01 -> 239d7862e2fe, Add last update field to newdle model
INFO  [alembic.runtime.migration] Running upgrade 239d7862e2fe -> 43f0bf77bf97, Add creator email and notify columns to newdle model
BEGIN;%0A%0A-- Running upgrade 93a638b96375 -> 679eab4aab01%0A%0AALTER TABLE participants ADD COLUMN comment VARCHAR DEFAULT '' NOT NULL;%0A%0AUPDATE alembic_version SET version_num='679eab4aab01' WHERE alembic_version.version_num = '93a638b96375';%0A%0A-- Running upgrade 679eab4aab01 -> 239d7862e2fe%0A%0AALTER TABLE newdles ADD COLUMN last_update TIMESTAMP WITHOUT TIME ZONE DEFAULT (now() at time zone 'utc')::timestamp NOT NULL;%0A%0AUPDATE newdles%0A SET last_update = final_dt%0A WHERE final_dt IS NOT NULL AND final_dt < now() AT TIME ZONE 'utc';;%0A%0AUPDATE alembic_version SET version_num='239d7862e2fe' WHERE alembic_version.version_num = '679eab4aab01';%0A%0A-- Running upgrade 239d7862e2fe -> 43f0bf77bf97%0A%0AALTER TABLE newdles ADD COLUMN creator_email VARCHAR DEFAULT '' NOT NULL;%0A%0AALTER TABLE newdles ADD COLUMN notify BOOLEAN DEFAULT 'false' NOT NULL;%0A%0AALTER TABLE newdles ALTER COLUMN notify SET DEFAULT 'true';%0A%0AUPDATE alembic_version SET version_num='43f0bf77bf97' WHERE alembic_version.version_num = '239d7862e2fe';%0A%0ACOMMIT;
```